lib/subscription-connection.js

```diff
@@ -171,18 +171,50 @@ module.exports = class SubscriptionConnection {
       }
     }
 
+    const document = typeof query !== 'string' ? query : parse(query)
+
+    if (!document) {
+      throw new Error('Must provide document.')
```

Comment:
We should probably define a specific Mercurius error here. Also, is this covered in the tests? I don't see a test case for it here, unless I'm missing something?

Reply:
Well, seems it's covered by tests right now, 'Must provide document.' is expected in 1 test. Anyway, this is the exact same error message thrown by the graphql.js lib itself... https://github.com/graphql/graphql-js/search?q=%22Must+provide+document%22

I don't know, maybe we can define a specific one for Mercurius, if you have any specific suggestion... otherwise I would need to analyse the flow a bit more.

Reply:
Cool, if it's covered by existing tests then all good for me - I must've missed it in the file diff